src/passes/Asyncify.cpp

```diff
@@ -1555,6 +1561,28 @@ struct Asyncify : public Pass {
                                          Builder::Mutable));
   }
 
+  void addImportedGlobals(Module* module) {
+    Builder builder(*module);
+    {
+      auto asyncify_state = builder.makeGlobal(ASYNCIFY_STATE,
+                                               Type::i32,
+                                               builder.makeConst(int32_t(0)),
+                                               Builder::Mutable);
+      asyncify_state->module = "env";                                         
```

Comment:
```suggestion
      asyncify_state->module = ENV;                                         
```
I believe there is a global string for this.